src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  justify-content: center;
+
+  padding: $paddingY $paddingX;
+  row-gap: $gapX;
+  column-gap: $gapY;
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, auto);
```

Comment:
I think you can skip the usage of repeat for 1 and just use auto.

Reply:
if I delete the 1, it won't work

Reply:
Did you try like this; `grid-template-columns: auto;`